lib/iris/coords.py

```diff
@@ -1478,32 +1619,74 @@ def _repr_other_metadata(self):
             result += ', circular=%r' % self.circular
         return result
 
-    @property
-    def points(self):
-        """The local points values as a read-only NumPy array."""
-        return self._points
-
-    @points.setter
-    def points(self, points):
-        points = np.array(points, ndmin=1)
```

Comment:
Oh I see, but the problem is this will promote points to have at least one dimension, so for a coord of length one you can assign as single value. Without this line that will fail at line 1632